velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -747,6 +747,39 @@ RowVectorPtr VectorFuzzer::fuzzRowChildrenToLazy(RowVectorPtr rowVector) {
       std::move(children));
 }
 
+RowVectorPtr VectorFuzzer::fuzzRowChildrenToLazy(
+    RowVectorPtr rowVector,
+    const std::vector<column_index_t>& columnsToWrapInLazy) {
```

Comment:
Is there an assumption that columns here appear in sorted order?

Reply:
Yes, I mentioned that in the method comment in the header file